src/IMangrove.sol

```diff
@@ -79,6 +79,15 @@ interface IMangrove is HasMgvEvents {
     external
     returns (uint takerGot, uint takerGave, uint bounty, uint feePaid);
 
+  function marketOrderByLogPrice(
+    OLKey memory olKey,
+    int maxLogPrice,
+    uint fillVolume,
+    bool fillWants,
+    uint maxGasreqForFailingOffers,
+    uint maxRecursionDepth
```

Comment:
It depends on how cautious  we are. If we are confident we cannot hit the 75 limit, then we can remove the override. Remember the limit only really affects successful offers since failing offers can also be controlled by maxGasreqForFailingOffers.

Reply:
I think that even if the 75 limit is wrong, it's fine if governance changes it, right? 

I think that `maxRecursionDepth` works differently from `maxGasreqForFailingOffers`, which has to be adjusted depending on type of clog. Whereas `maxRecursionDepth` ideally should just be set to the perfect value and never changed?